Products/GenericSetup/tool.py

```diff
@@ -921,17 +921,30 @@ def getProfileInfo(self, profile_id):
 
     @security.protected(ManagePortal)
     def getDependenciesForProfile(self, profile_id):
-        if profile_id is None:
-            return ()
-        if profile_id.startswith("snapshot-"):
-            return ()
```

Comment:
I don't know why the early exit here is wrong:  without a `profile_id` we cannot look up dependencies, and snapshots never have dependencies (they are full exports of a site's state, not intended to be composed with others).

Reply:
It's not wrong, just wanted to save two lines. But yes, saving processing time is better, thanks.